citesphere/src/main/webapp/WEB-INF/views/auth/group/item.html

```diff
@@ -360,20 +430,68 @@ <h2 style="margin-top: 0">
+                        <span class="additional-item" th:if="${additionalFiles?.filename}">
+                             <span th:id="|additionalFiles${status.index}|" class="additional-item" th:data-viewupload-uploadId="${upload.uploadId}" th:additionalFiles-href="@{|/auth/group/${zoteroGroupId}/items/${citation.key}/giles/${additionalFiles?.id}|}" th:data-viewupload-additionalFiles-filename="${additionalFiles?.filename}" >
+					 	     </span>
+                        </span>
+                        </th:block>
+                        </span>
+                        </span> 
+                </th:block>
+                <a class="view-link">[[${upload.uploadedFile?.filename}]]</a>
+	        </span>
 	        </span>
 	        <span th:unless="${upload.documentStatus != null && (upload.documentStatus != T(edu.asu.diging.citesphere.model.bib.GilesStatus).SUBMITTED || T(edu.asu.diging.citesphere.model.bib.GilesStatus).NEW)}">
 	        File is being processed... ([[${upload.progressId}]])
 	        </span>
 	        <span th:if="${upload.documentStatus == T(edu.asu.diging.citesphere.model.bib.GilesStatus).FAILED}" class="label label-danger">Failed [[${upload.progressId}]]</span>
 	        <span th:if="${upload.documentStatus == T(edu.asu.diging.citesphere.model.bib.GilesStatus).NEW}" class="label label-default">In Queue</span>
 	        <span th:if="${upload.documentStatus == T(edu.asu.diging.citesphere.model.bib.GilesStatus).SUBMITTED}" class="label label-default">Processing</span>
-	        </li>
+	        
+	        
+	       </li>
         </ul>
-        
+
+	    <div class="modal fade" id="viewuploadModal" 
+			tabIndex="-1" role="dialog"
+			aria-labelledby="viewUploadModal" aria-hidden="true">
+			<div class="modal-dialog" role="document">
+            <div class="modal-content" id="viewUploadContent">
+               <div class="modal-header">
+                   <h4 class="modal-title" id="viewUploadModal">Files uploaded to and extracted by Giles
+                    <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                        <span aria-hidden="true">×</span>
+                    </button>
+                    </h4>
+                </div>
+                <div class="modal-body" id="viewUploadBody">
+                  <ul class="list-group" >
+                  	 <li><p style="margin-bottom:0;"> Upload File: 
+                     <a id="uploadedFileName"> </a>
+                     </p></li>
+                     <li><p style="margin : 0; padding-top:0;"> Extracted Text: 
+                     <a id="extractedFileName"> </a>
+                     </p></li>
+                   </ul>
+                </div>
+                <div class="modal-footer">
+                    <button type="button" class="btn btn-secondary" data-dismiss="modal"> Close </button>
+                </div>
+            </div>
+       </div>
+    </div>
+
```

Comment:
I think all of these can be simplified by just creating a model for each upload and then show/hide those modals when the user clicks on an upload in the list. There is not javascript really needed (beyond opening/closing the modal).